package.json

```diff
@@ -54,10 +54,14 @@
   },
   "homepage": "http://ftrack.com",
   "dependencies": {
+    "isomorphic-ws": "^5.0.0",
     "loglevel": "^1.8.1",
     "moment": "^2.29.4",
     "uuid": "^9.0.0"
   },
+  "peerDependencies": {
+    "ws": "^8.13.0"
+  },
```

Comment:
add [peerDependenciesMeta](https://yarnpkg.com/configuration/manifest#peerDependenciesMeta) config to tell that ws is optional as well to not output warnings when installing